src/Config/Level/CodeQualityLevel.php

```diff
@@ -42,7 +42,6 @@
 use Rector\CodeQuality\Rector\FuncCall\SimplifyStrposLowerRector;
 use Rector\CodeQuality\Rector\FuncCall\SingleInArrayToCompareRector;
 use Rector\CodeQuality\Rector\FuncCall\UnwrapSprintfOneArgumentRector;
-use Rector\CodeQuality\Rector\FunctionLike\SimplifyUselessVariableRector;
```

Comment:
please remove the list in line 116 as well

Reply:
this line should be removed as well on registering rule, not only use statement:

https://github.com/rectorphp/rector-src/blob/ed6b9124d64a80b10e6e81153353b8e59cd1d8e1/src/Config/Level/CodeQualityLevel.php#L117

on your PR, it on line 116 as removed one line